deploy/cassandra_restore_copy.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
```

Comment:
Can we rename the file? To something like suffixed with v2?

Reply:
sure, I added copy because it's using copy paste approach.

Reply:
Name resolved to v2.py